INFO: 2018-08-09 18:54:11,719:DartRunner.py:108: Logfile is /root/Dart/log/DartRunner-2018-08-09-185411.log
INFO: 2018-08-09 18:54:11,721:DartRunner.py:109: Current Process Id: 43682
INFO: 2018-08-09 18:54:11,724:DartRunner.py:170: clusterList = DartExe 
INFO: 2018-08-09 18:54:11,725:DartRunner.py:178: DartExe
INFO: 2018-08-09 18:54:11,729:DartRunner.py:1974: The Json File may have syntax issues! Check the config file DartExe.cfg!
INFO: 2018-08-09 18:54:11,730:DartRunner.py:1975: Expecting property name: line 12 column 6 (char 325)
INFO: 2018-08-09 18:54:11,731:DartRunner.py:1976: Exiting Program!
ERROR: 2018-08-09 18:54:11,731:DartRunner.py:1977: (<type 'exceptions.ValueError'>, ValueError('Expecting property name: line 12 column 6 (char 325)',), <traceback object at 0x7f4fb7092320>)
ERROR: 2018-08-09 18:54:11,749:DartRunner.py:1978: Traceback (most recent call last):
  File "DartRunner.py", line 1961, in loadClusterConfigs
    cfgJson = json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 12 column 6 (char 325)

